resources/public/api/conf/2.0/examples/broughtForwardLosses/retrieveBFLoss/retrieve_fhl_response.json

```diff
@@ -5,17 +5,17 @@
   "lastModified": "2018-07-13T12:13:48.763Z",
   "links": [
     {
-      "href": "/individuals/losses/TC663795B/brought-forward-losses/1234568790ABCDE",
+      "href": "/individuals/losses/TC663795B/brought-forward-losses/XAIS12345678910",
```

Comment:
If you are using XAIS12345678910 for the businessId then I would suggest you use a different value for the lossId. Try to be consistent across the endpoints if possible please. I think 1234568790ABCDE is used in the lossId parameter description.